ember-cli-build.js

```diff
@@ -26,12 +56,13 @@ module.exports = function(/* defaults */) {
     destDir: '/tests'
   });
 
-  var srcJSHint = new JSHint(src);
+  var srcJSHint = new JSHint(srcJS);
   var testsJSHint = new JSHint(tests);
+  var srcTSLint = new TSLint(srcTS);
 
-  var allTests = concat(new MergeTrees([tests, srcJSHint, testsJSHint]), {
+  var allTests = concat(new MergeTrees([tests, srcJSHint, testsJSHint, srcTSLint]), {
     outputFile: '/tests/tests.js',
-    inputFiles: ['**/*.js'],
+    inputFiles: ['**/*.js', '**/*.ts'],
```

Comment:
This seems somewhat odd?  Doesn't TSLint emit JS files?  If we really put typescript into a `.js` file for the browser, it will throw parse errors...